rope/base/worder.py

```diff
@@ -341,7 +353,7 @@ def is_import_statement(self, offset):
         line_start = self._get_line_start(last_import)
         return (
             self._find_import_end(last_import + 7) >= offset
-            and self._find_word_start(line_start) == last_import
+            and self._find_next_word_start(line_start) == last_import
```

Comment:
Well, this didn't break any tests, and has fixed the ones I introduced. I'm still not sure I understood the original code correctly though